code/sql/algoliasearch_setup/mysql4-upgrade-1.5.0-1.6.0RC.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
```

Comment:
We need to be careful with the filename, because I am not sure what happens when you release 1.6.0